### ISSUE TITLE
request for color option for the complete signal wave

### ISSUE THREAD

**Issue by dkgoyal** [wavedrom/wavedrom#256]
thank you very much for creating this awesome tool for waveform generation. 

It will be very helpful if we can color a signal waveform with some option . 
That feature will be very useful.

**Comment by drom** [comment]
@dkgoyal are you thinking about line, background, fill or text colors?
Could you provide some examples that would be useful?

**Comment by grigoraghanyan** [comment]
I agree with enhancement request. We are using wavedrom as one of options to visualize waveforms in our reports and there is need to visually (by color)  mark some signals (both bus and bit) and also have waveforms colored with different color (red is preferable) starting from the frame (or only at the frame) where FAIL happens. Also please make the available colors more visually "non-neutral". It would be better to have exact Red, Green and Yellow colors instead of supported ones.

**Comment by drom** [comment]
@grigoraghanyan the color scheme can be changed using skin mechanism described here: https://github.com/wavedrom/wavedrom/blob/master/unpacked/README.md
For other ways of coloring individual bits of a diagram, I would need some case by case visual examples to proceed.

**Comment by raulbehl** [comment]
I think #271 implements the needed support for this? 

@drom : Here is what gtkwave allows you to do and I believe this is what was requested here: 
![Screenshot from 2020-12-11 20-00-07](https://user-images.githubusercontent.com/7036260/101937077-63b25000-3c07-11eb-82eb-8e6cc6123183.png)
